File: dayThree/morningSession/java/FaceBookLikeSystem.java
public class FaceBookLikeSystem{
  
    public static String textDisplay(String [] names){
        
        int nameLength = names.length;

        if (nameLength == 0 ){
            return "no one likes this";
        }
        else if(nameLength == 1){
             String result = String.format("""
             %s likes this"""
                                            ,names[0]);
            return result;
        }
        else if(nameLength == 2){
             String result = String.format("""
             %s and %s likes this"""
                                            ,names[0],names[1]);
            return result;
        }
        else if(nameLength == 3){
            String result = String.format("""
             %s, %s and %s likes this"""
                                            ,names[0],names[1],names[2]);
            return result;
        }
        else if(nameLength >= 4){
             String result = String.format("""
             %s, %s and %d others likes this"""
                                            ,names[0],names[1], nameLength-2);
            return result;
        }
        return null;
    }
}

File: dayThree/morningSession/python/faceBook_like_system.py
def text_display(names):
    name_length = len(names)

    if (name_length == 0 ):
        return "no one likes this"
        
    elif(name_length == 1):
        return (f"{names[0]} likes this}")

    elif(name_length == 2):
        return (f"{names[0]} and {names[1]} likes this}");
        
    elif(name_length == 3):
        return (f"{names[0]}, {names[1]} and {names[2]} likes this}");
        
    elif(name_length >= 4):
        return (f"{names[0]}, {names[1]} and {names_length - 2} likes this}");
        }
    return null;
